### ISSUE TITLE
Populate Embedded Multiple input with dropdown

### ISSUE THREAD

**Issue by Lordfef** [unclead/yii2-multiple-input#101]
Excuse me for new issue but i'm working on this part of my project.
As you can see in my other issue #100  i'm working on complex embedded multiple item.
This is a part of Post Request i get on submit:
```
...
    'wgs' => [
        0 => [
            'id_group' => '12'
            'name' => 'XXX'
            'subject' => [                        <<'type' => MultipleInput::class
                'id_subject' => [                << this is my column TYPE_DROPDOWN
                    0 => '125'
                    1 => '126'
                ]
            ]
        ]
     
    ]
...
```

Now i want to edit this model and i create a function to populate it. And i generate a model like this:

```
...
	 [wgs] => Array ( 
		[0] => Array ( 
			[id_group] => 12 
			[name] => XXX 
			[subject] => Array ( 
				[id_subject] => Array ( 
					[0] => 125 
					[1] => 126 
				) 
			) 
		) 
...
```
In my view i can see well [id_group] and [name] with model values, but my embedded multiple input is not populated. What i'm missing?
Thank's

**Comment by unclead** [comment]
Embedding is a new feature and when i could cover only basic use cases. If you can provide me the code to reproduce the bug (e.g. on Gist) it'd be great

**Comment by Lordfef** [comment]
i put model-controller action-view here: https://gist.github.com/Lordfef/b963813327cce6dca3d626486a41773c

**Comment by unclead** [comment]
I can't find s simple way to fix an issue. But i suggest you use the following workaround

```
 $calc2 = 0;
            foreach ($groupsubjects as $gs) {
                $wgs->wgs[$calc]['subject'][$calc2]['id_subject'] = $gs->id_subject;
                $calc2++;
            }
```

`wgs[$calc]['subject'][$calc2]['id_subject']` insread of `wgs[$calc]['subject']['id_subject'][$calc2]`

Embedding is so weak feature and has many pitfalls

**Comment by Lordfef** [comment]
the problem is that anyway the next rows i create are all called with the same name and the js stop to work.

**Comment by unclead** [comment]
Hmmm it works for even for new rows... http://joxi.ru/4AkpVj9sMd19xr
I'm sorry but i don't have free time to reproduce the bug. 
You can create a repo with based on basic template which i can you to reproduce the bug.

this is how i populate model

```
   public static function PopulateWGS()
    {
        $wgs = new WizGroupSubject();
        $calc= 0;
        $groups = [
            1 => 'Group1',
            2 => 'Group2',
            3 => 'Group3',
        ];
        foreach ($groups as $group_id => $group_name) {
            $wgs->wgs[$calc]['id_group'] = $group_id;
            $wgs->wgs[$calc]['name'] = $group_name;
            $groupsubjects = [
                2,3

            ];
            $calc2 = 0;
            foreach ($groupsubjects as $gs) {
                $wgs->wgs[$calc]['subject'][$calc2]['id_subject'] = $gs;
                $calc2++;
            }
            $calc++;
        }
        return $wgs;
    }
```

**Comment by Lordfef** [comment]
Thank's @unclead , for now using: $wgs->wgs[$calc]['subject'][$calc2]['id_subject'] = $gs;
i can populate well my embedded multiple input.

**Comment by Lordfef** [comment]
Now that i could populate my embedded multiple input i have a javascript error when i add new (master) row.

file: jquery.multipleInput.js (line 306, col 9)
TypeError: data is undefined
error source line: form.yiiActiveForm('add', $.extend({}, data.attributeDefaults, {

**Comment by unclead** [comment]
i can't reproduce a bug in my test project.
> You can create a repo with based on basic template which i can use to reproduce the bug.

**Comment by unclead** [comment]
@Lordfef will you help me and prepare a repo to reproduce a bug? 😄

**Comment by Lordfef** [comment]
hi @unclead i'm very pleased to help you, but i'm very newbie with public git, i use git only for version control of my projects. Can you suggest me the right way to prepare a repo for you?

**Comment by unclead** [comment]
Just create new project based on basic template and commit it into a new project at github.
I'll be able to spent time on this issue at the weekend

**Comment by unclead** [comment]
probably i fixed it in scope of 3b597aff45a9dabd92b99846d023ca3c37c10a17
